mypyc/lib-rt/CPy.h

```diff
@@ -1277,6 +1312,10 @@ static void CPy_Reraise(void) {
     PyErr_Restore(p_type, p_value, p_traceback);
 }
 
+static int CPy_NoErrOccured(void) {
```

Comment:
I was a bit worried about the extra call required, but turns out that this is rarely called so the performance impact will be negligible.